src/app.js

```diff
@@ -1 +1,44 @@
 'use strict';
+
+const readline = require('readline');
+const fs = require('fs');
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+let fileName = '';
+let fileContent = '';
+const reassignFileName = function(userInput) {
+  fileName = userInput;
+};
+const reassignFileContent = function(userInput) {
+  fileContent = userInput;
+};
+
+const askQuestion = (question, func) => {
+  return new Promise((resolve, reject) => {
+    rl.question(question, (userInput) => {
```

Comment:
don't cut variable name. Use full naming

Reply:
Did you mean "rl"?